docs/troubleshooting.md

```diff
@@ -217,3 +217,81 @@ The following is a simple example of this occurring:
+| net_scrape_update                                | Time spent updating the internal model with information read from /proc (set removed entries as inactive, update activity timestamp) |
+| net_fetch_state                                  | Time spent to build a delta message content (connections + endpoints) to send to Sensor                                              |
+| net_create_message                               | Time spent to serialize the delta message and store the resulting state for next computation.                                        |
+| net_write_message                                | Time spent sending the raw message content.                                                                                          |
+
+
+### ColletorStats counters -> rox_collector_counters
+
+| Name                                             | Description                                                                                                                          |
+|--------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------|
+| net_conn_updates                                 | Each time a connection object is updated in the model (scrapes, and kernel events).                                                  |
+| net_conn_deltas                                  | Number of connection events sent to Sensor.                                                                                          |
+| net_conn_inactive                                | Accumulated number of connections destroyed (closed)                                                                                 |
+| net_cep_updates                                  | Each time an endpoint object is updated in the model (scrapes only).                                                                 |
+| net_cep_deltas                                   | Number of endpoint events sent to Sensor.                                                                                            |
+| net_cep_inactive                                 | Accumulated number of endpoints destroyed (closed)                                                                                   |
+| net_known_ip_networks                            | Number of known-networks defined.                                                                                                    |
+| net_known_public_ips                             | Number of known public addresses defined.                                                                                            |
+| process_lineage_counts                           | Every time the lineage info of a process is created (signal emitted)                                                                 |
```

Comment:
The idea of lineage is not explained anywhere, could you pls summarize it in a few words somewhere?